### ISSUE TITLE
Not Able to run Hello-world binary.

### ISSUE THREAD

**Issue by ishan-rep** [haskell-cryptography/HsOpenSSL#86]
Post stack build , when I run the hello-world binary, it is throwing:

```
cipher: DES-CBC
generating RSA keypair...
plain text to encrypt: Hello, world!

encrypting...
hello-world: user error (error:0308010C:digital envelope routines::unsupported)
```

I am running this on Mac M1 machine (not sure if this could be a reason).

**Comment by ishan-rep** [comment]
des <- liftM fromJust $ getCipherByName "DES3"

This works on my system : changed from "DES-CBC".

**Comment by vshabanov** [comment]
Fixed with your #87 and available in HsOpenSSL-0.11.7.7
